### DISCUSSION TITLE
Usage with Nightshade?

### DISCUSSION THREAD

**Discussion by LoosfDoesThings** [lshqqytiger/ZLUDA#2] [Ideas]
Hello. Nightshade is a useful resource for visual artists trying to protect their work, and it can run on CPU,
 but with the CUDA acceleration, image processing goes from 30 minutes of cpu in full load to an estimated of 5 minutes on the GPU

It makes use of PyTorch, so assumed that was the issue with the base version of ZLUDA

I have been trying to utilize it, the first issue was relatively easily solved, requiring  hipRTC dll, which I added after getting the SDK.
![imagen](https://github.com/lshqqytiger/ZLUDA/assets/161077164/f909f3a7-f1b9-4353-9367-ca51b617d3e6)

But even after that, the behavior is the same as the  base version of ZLUDA, in which  it will attempt to function, but immediately it will close itself.

![imagen](https://github.com/lshqqytiger/ZLUDA/assets/161077164/398f8d48-6282-4a83-b9a9-21f8f3a87237)

 https://nightshade.cs.uchicago.edu/whatis.html

**Comment by lshqqytiger** [comment]
Nightshade has similar structure with Glaze. So I can run Glaze after replacing some dll files with ZLUDA's. https://github.com/vosen/ZLUDA/issues/150
But Nightshade doesn't work as I tried.
